packages/middleware-joi/src/schema.ts

```diff
@@ -0,0 +1,15 @@
+const Joi = require('joi');
```

Comment:
*ESModule* imports would be more consistent with rest of the code:
`import * as Joi from 'joi';`